podcasts/Episode Info Coordinator/ShowInfoCoordinator.swift

```diff
@@ -59,7 +64,10 @@ actor ShowInfoCoordinator: ShowInfoCoordinating {
         guard let transcripts = metadata?.transcripts else {
             return []
         }
-
+        if let transcript = Episode.Metadata.TranscriptFormat.bestTranscript(from: transcripts),
+           let url = URL(string: transcript.url) {
+           let _ = try? await transcriptDataRetriever.loadTranscript(url: url)
```

Comment:
My understanding is that we wanted to have the transcript file the moment we downloaded an episode, If I understood correctly the codebase this was the place to be used. 
Do you only get shown notes and artwork info if the user opens the episode screen?
Another advantage of this is the transcript is just ready when the user taps on it.

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/1955/commits/c941afbb889fbe628af01062f9b8c525acc4a1c0

Reply:
>Do you only get shown notes and artwork info if the user opens the episode screen?

Before, yes. Show notes would only display if downloading or checking them. Artwork info is also only cached (the image itself) if downloading or playing the episode.

This has changed though because the same endpoint that returns the show notes also return artwork, transcript, etc, etc.

Reply:
Ok now we only download transcripts metadata when opening the episode, but if we download the episode we will also cache the transcript locally so it can be opened even if offline.